### ISSUE TITLE
--omit-add doesn't exist

### ISSUE THREAD

**Issue by jrabbit** [CleanCut/green#231]
docs say it should exist
```
  -O, --clear-omit      Green tries really hard to set up a good list of patterns of files to omit from coverage reports. If the default
                        list catches files that you DO want to cover you can specify this flag to leave the default list empty to start
                        with. You can then add patterns back in with --omit-add. The default list is something like
                        '*/test*,*/mock*,*(temp dir)*,*(python system packages)*' -- only longer.
```
bad behavior looks like:
```
└> green -r --omit-add bottle
usage: green [options] [target [target2 ...]]
green: error: unrecognized arguments: --omit-add
```

**Comment by jrabbit** [comment]
So is it not possible to extend the list I have to replace it entirely?

**Comment by CleanCut** [comment]
Thanks! I fixed the docs. It should have mentioned `-o, --omit-patterns`.  The documentation fix will go out in the next release.

> So is it not possible to extend the list I have to replace it entirely?

Sure you can, just use `-o, --omit-patterns` without passing in `-O, --clear-omit`

**Comment by jrabbit** [comment]
I'll have to make sure that works, I was struggling with that and might have tried it but will confirm today hopefully.

**Comment by CleanCut** [comment]
Fix is included in 3.2.2, just released.
